install-and-start-cbd.sh

```diff
@@ -277,6 +277,11 @@ aws_prepare_os_user() {
     echo "cloudbreak ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/cloudbreak
 }
 
+non_aws_prepare_os_user() {
+    echo "# This is generated by the Cloudbreak deployment's cloud-init script" >> /etc/sudoers.d/cloudbreak
+    echo "cloudbreak ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/cloudbreak
```

Comment:
This is not necessary as the configured user get root permissions on the created Azure VM.

The other files of the PR are only generated by the make command so I will close this PR.